components/monitoring/base/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - install_prometheus.yaml
```

Comment:
Installation failed with -> `Failed, InterOperatorGroupOwnerConflict, intersecting operatorgroups provide the same apis`

Reply:
cannot see anymore